apps/api/src/app/integrations/dtos/create-integration-request.dto.ts

```diff
@@ -1,11 +1,27 @@
-import { IsBoolean, IsDefined, IsEnum, IsString, ValidateNested } from 'class-validator';
+import { IsBoolean, IsDefined, IsEnum, IsMongoId, IsOptional, IsString, ValidateNested } from 'class-validator';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
 import { ChannelTypeEnum, ICreateIntegrationBodyDto } from '@novu/shared';
+
 import { CredentialsDto } from './credentials.dto';
-import { ApiProperty } from '@nestjs/swagger';
-import { Type } from 'class-transformer';
 
 export class CreateIntegrationRequestDto implements ICreateIntegrationBodyDto {
```

Comment:
The only required fields now are: `providerId, channel`. 
The default values for the boolean flags are `active: false, check: true`.